lib/features/caching/config/fcm_isolate_name_server.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:ui' as flutter;
+
+import 'package:hive_ce/hive.dart';
+
+class FcmIsolateNameServer extends IsolateNameServer {
```

Comment:
it uses for what?

Reply:
It is a mechanism for sharing ports between isolates. `IsolateNameServer` allows registering and looking up a SendPort by name, allowing the secondary isolate to connect to the primary isolate or vice versa without passing the port through the constructor.